drupal/drush/sites/self.site.yml

```diff
@@ -0,0 +1,51 @@
+# Docs at https://github.com/drush-ops/drush/blob/master/examples/example.site.yml
+
```

Comment:
Should there be Lando specific environment here as well?

Reply:
The `lando drush` command is the correct way to run drush inside lando. There should generally be no need to run drush on the host.

Reply:
Does this also work for running drush commands on remote servers?